### ISSUE TITLE
Missing Units from get_power_apsim_met

### ISSUE THREAD

**Issue by wentama** [femiguez/apsimx#152]
Hi, for some reason whether I use `get_power_apsim_met` and write it to met using `get_power_apsim_met`, the met file is always missing the units in row 9: '() () (MJ/m2/day) (oC) (oC) (mm) (%) (m/s)'. It could be manually fixed if I go in and paste the units after the file is created but just wondering what could be wrong. Thanks.

This is the code:
![image](https://github.com/femiguez/apsimx/assets/107167742/2f4d6159-7864-44c7-af44-d00c09b1bf33)

This is the output met file:

![image](https://github.com/femiguez/apsimx/assets/107167742/42d89b3b-bafb-41a2-ad69-0b21762025fe)

**Comment by wentama** [comment]
@femiguez thanks for adding the check constraint! I attempted to use the updated apsimx R package to load the weather data and I'm now getting the message from `check_apsim_met` regarding the missing units from `get_power_apsim_met` (shown below). Just wondering why were the units auto-filled the first time I used `get_power_apsim_met` maybe 2 months ago, but now the units are missing for `get_power_apsim_met`? I also don't recall syncing apsimx with any new commits during that time.

![image](https://github.com/femiguez/apsimx/assets/107167742/451182bc-4dbd-46b8-9cae-938846ebef80)
